.github/workflows/ci_eval.yaml

```diff
@@ -58,5 +59,51 @@ jobs:
             -e "git+https://github.com/iree-org/iree-turbine.git#egg=iree-turbine"
           pip install --no-compile -r requirements.txt -r sharktank/requirements-tests.txt -e sharktank/
 
-      - name: Run perplexity test
-        run:  pytest -n 4 -v -s sharktank/tests/evaluate/perplexity_test.py  --longrun
+      - name: Run perplexity test with vmfb
+        run:  pytest -n 4 -v -s sharktank/tests/evaluate/perplexity_vmfb_test.py --iree-device='hip://7' --longrun
+
+  # test_perplexity_torch:
```

Comment:
Will uncomment this job once I test vmfb version, as the jobs run sequentially in unordered fashion.